bin/commands/stack/list.js

```diff
@@ -0,0 +1,12 @@
+
+
+module.exports = {
```

Comment:
This is actually a dummy placeholder and will likely be deleted/removed. Ignore.